apps/mull-ui/src/app/components/time-slider/time-slider.tsx

```diff
@@ -0,0 +1,52 @@
+import Slider from 'rc-slider';
```

Comment:
Why isn't the package.json updated? If you add a package to the project, make sure it's in the package.json

Reply:
hmmm, not sure why it's not there because I do remember having a package commit. Added it